docs/documentation/tabs-explained/tab-summary/README.md

```
@@ -7,3 +7,10 @@ The top section of the page provides information about the selected location suc
 The bottom section of the page comprises the heating and cooling degree day chart and four violin plots showing the distribution of the dry-bulb air temperature (Tdb), relative humidity (RH), Global Horizontal Irradiance (GHI), and wind speed (U).
 
 ![Tab summary top](../../../.gitbook/assets/clima-summary-bottom.png)
+
+### Video Tutorial
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)